smg-automotive/components-pkg
# feat: added regular section variation

References https://autoricardo.atlassian.net/browse/DM-358

## Motivation and context

Add regular section variation

## Before

No regular section variation 😱

## After

Regular section variation 🥳

![Screenshot 2022-07-21 at 11 09 13](https://user-images.githubusercontent.com/37380787/180176411-8fb01381-9c6a-49df-b4bd-c7ac51faeca1.png)

## How to test

Please doble check the section variations: hero and regular. Use the branch locaaly.

## Thank you 🌵
